server/src/helpers/utils.helper.ts

```diff
@@ -29,3 +30,11 @@ export async function getCachedConnection(dataSourceId, dataSourceUpdatedAt): Pr
     }
   }
 }
+
+export function sanitizeInput(value: string) {
+  return sanitizeHtml(value, {
+    allowedTags: [],
```

Comment:
This would not allow any tags to be entered, might conflict with the html widget, probably add a default list here?

Reply:
What do you suggest? There would be HTML as input?

Reply:
Yeah, like you said since appDef is not being affected we can ignore it for now,